dir.props

```diff
@@ -274,6 +274,10 @@
 
   <!-- Setup properties per OSGroup -->
   <Choose>
+    <When Condition="'$(IsReferenceAssembly)' == 'true' OR'$(MSBuildProjectExtension)' == '.pkgproj'">
```

Comment:
I want PKGPROJS to define their runtime explicitly.  For reference assemblies it was causing a problem because some CSProjs set OSGroup and that flows to the reference assemblies.  We validate that Reference assemblies never set a RID, so this causes failures.